README.md

```diff
@@ -1,8 +1,29 @@
+### ToRoman
+[![Build Status](https://cloud.drone.io/api/badges/nytimes/library/status.svg)](https://cloud.drone.io/nytimes/library) 
+
+
+**Table of Contents**
+
+- [Making a Contribution](#contribution)
+- [Features](#🚀Features)
+- [Installation](#📦Installation)
+- [Usage](#📥Usage)
+    - [Convert Integer to Roman Numerals](#convertintegertoromannumerals)
+     - [Convert  Roman Numerals to Integer](#convertromannumeralstointeger)
+- [Found this project useful](#Foundthisprojectuseful)
 # toroman
 
 A minimalist library for Roman numeral operations.
+A Minimal Library for Roman Numeral Operations.
+
+## CONTRIBUTION
+
+I know you find this project amazing and beautiful i know that feeling there but Please do take your time to 
+read our contributing.md docs first before you make your first contribution.
 
-## 🚀 Features
+oya abeg read our contributing.md docs first before you push rubbish.
```

Comment:
Why?

Reply:
just for fun let me remove that and leave the top one